pkg/server/testdata/machine-configs/test-config.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     - nosmt
   config:
     ignition:
+      version: "2.2.0"
```

Comment:
Question: did this work before without an explicit ignition version?

Reply:
yes, this wasn't caught by tests before